<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdModificadores</class>
 <widget class="QWidget" name="wdModificadores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Zekton</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Variante:</string>
      </property>
      <property name="buddy">
       <cstring>leNombre</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="ifRefExt" name="leVariante">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ifDesplegableExt" name="cbVariante">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Zekton</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Referencia:</string>
      </property>
      <property name="buddy">
       <cstring>leRef</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="ifCadena" name="leRef">
      <property name="maximumSize">
       <size>
        <width>663</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Zekton</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre: </string>
      </property>
      <property name="buddy">
       <cstring>leNombre</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="ifCadena" name="leNombre">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Zekton</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Orden:</string>
      </property>
      <property name="buddy">
       <cstring>leNombre</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="ifCadena" name="leOrden">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>leRef</zorder>
   <zorder>leNombre</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>leOrden</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ifRefExt</class>
   <extends>QLineEdit</extends>
   <header>Interfaz/ifCampos.h</header>
  </customwidget>
  <customwidget>
   <class>ifCadena</class>
   <extends>QLineEdit</extends>
   <header>Interfaz/ifCampos.h</header>
  </customwidget>
  <customwidget>
   <class>ifDesplegableExt</class>
   <extends>QComboBox</extends>
   <header>Interfaz/ifCampos.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leVariante</tabstop>
  <tabstop>cbVariante</tabstop>
  <tabstop>leRef</tabstop>
  <tabstop>leNombre</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
